src/rust/zh.mangabz/res/source.json

```
@@ -3,7 +3,7 @@
 		"id": "zh.mangabz",
 		"lang": "zh",
 		"name": "Māngabz",
-		"version": 2,
+		"version": 3,
 		"url": "https://mangabz.com/",
```

looks like this slipped in by accident before, urls shouldn't have trailing slashes
```suggestion
		"url": "https://mangabz.com",
```

---

Done